src/sagemaker/image_uris.py

```diff
@@ -216,6 +218,9 @@ def retrieve(
     else:
         tag_prefix = version_config.get("tag_prefix", version)
 
+    if repo == f"{framework}-inference-graviton":
+        container_version = f"{container_version}-sagemaker"
+
```

Comment:
Question, I believe this would cause v1.0-sagemaker versus sagemaker-v1.0 Am I correct? if so, this will need to be reversed.
"1.12.1-cpu-py38-ubuntu20.04-sagemaker-v1.0"